### ISSUE TITLE
Add time zone (DST) support for scheduled triggered webjobs

### ISSUE THREAD

**Issue by HansOlavS** [projectkudu/kudu#1783]
Right now scheduled triggered webjobs are using either Azure Scheduler or NCronTab with CRON-expressions and currently none of these support time zones or DST (Daylight Saving Time).

My issue is that I have a lot of jobs across many web apps set up to run at specific times of day, like 15:00 each day. And since I live in a timezone with DST all my schedules need to be manually edited each time summer-time or winter-time comes.

How could we fix this? I'm considering adding support for time zone and DST in the NCronTab and trying to create a pull-request for that (not sure how much work that would be, but I think it's doable).

Thoughts, anyone?

Interesting StackOverflow article: http://stackoverflow.com/questions/13195999/daylight-savings-and-cron

Another dude: http://codeofmatt.com/2013/11/04/windows-azure-scheduler/

Cheers!

**Comment by mathewc** [comment]
Anything we discuss/decide is also relevant to **TimerTrigger** in the [WebJobs SDK repo](https://github.com/Azure/azure-webjobs-sdk-extensions) - it also uses NCronTab.

**Comment by ahmelsayed** [comment]
you can potentially [change the time zone for a site](http://blogs.msdn.com/b/tomholl/archive/2015/04/07/changing-the-server-time-zone-on-azure-web-apps.aspx) from UTC, to whatever time zone you want using the `WEBSITE_TIME_ZONE` app setting, then let Windows deal with DST for you.

**Comment by HansOlavS** [comment]
Thanks, @ahmelsayed, that will actually solve my problem! Sweet!

**Comment by HansOlavS** [comment]
Damn I love Azure and App Service. It's pure gold.

**Comment by rakeshshindeeternus** [comment]
I have also written one web job ...which was running in UTC environment where as I want it to run in IST .. I just  add application setting "WEBSITE_TIME_ZONE" with value "India Standard Time" and restarted web app. its solve my problem..simply great Azure !!

**Comment by Porkechebure** [comment]
Issue is still on rampage.
Azure is in UTC. I tried to set in my local.settings.json, servicebustrigger function this:
WEBSITE_TIME_ZONE: "Greenwich Standard Time" or either "GMT Standard Time"

to simulate in local the azure timezone (UTC+0).

Nothing. It keeps adding 1 hour becuase we are in the summer daylight saving timezone.

The only way I found is removing 1 hour with .AddHours(-1).

No class or timezone setting is available to remove the extra hour.

Not good.
